docs/developer-guide/contributors-quickstart.md

```diff
@@ -24,11 +24,13 @@ Before starting, ensure you have the following tools installed with the specifie
 
 2. Clone the forked repository:
 ```shell
-mkdir -p $GOPATH/src/github.com/argoproj/
-cd $GOPATH/src/github.com/argoproj/
+mkdir -p FOLDER-OF-YOUR-CHOICE/github.com/argoproj/
+cd FOLDER-OF-YOUR-CHOICE/github.com/argoproj/
 git clone https://github.com/YOUR-USERNAME/argo-cd.git
 ```
 
+Please note that the local build process uses GOPATH and that path should not be used, unless the Argo CD repository was directly cloned in it.
```

Comment:
```suggestion
Please note that the local build process uses GOPATH and that path should not be used, unless the Argo CD repository was directly cloned in it already.
```

do we need this anymore? `FOLDER-OF-YOUR-CHOICE` depicts this statement already. WDYT?

Reply:
I think both are important and would keep them, but if anyone else feels strongly for your point, I will remove this part.